pkgs/development/libraries/mimalloc/minimal.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname   = "mimalloc-minimal";
+  version = "2.0.5";
+
+  src = fetchurl {
```

Comment:
Github fetcher doesn't work?

Reply:
this is in stdenv bootstrap

Reply:
Hum, well noted.

Reply:
Can you copy the bootstrapping comment from other packages, so that this question is not coming up again?

Reply:
Done